BraintreeCore/src/main/java/com/braintreepayments/api/ClientTokenCallback.java

```diff
@@ -0,0 +1,8 @@
+package com.braintreepayments.api;
+
+import androidx.annotation.NonNull;
+
+public interface ClientTokenCallback {
```

Comment:
Doc string

Reply:
Also wondering about the naming on this - Do we want to be consistent with `Authorization`? Seems like sometimes we use authorization and sometimes we use client token

Reply:
I went with ClientToken prefixes to encourage merchants to fetch a client token, although you can technically callback any valid form of Authorization with a `ClientTokenProvider`. I can add doc strings but maybe a second pass on the naming once everything's in place?